spring-jms/src/main/java/org/springframework/jms/core/JmsClient.java

```diff
@@ -67,6 +67,7 @@ interface JmsClient {
 	/**
 	 * Create a new {@code JmsClient} for the given {@link ConnectionFactory}.
 	 * @param connectionFactory the factory to obtain JMS connections from
+	 * @return a new {@code JmsClient} instance
```

Comment:
This is not strictly necessary.

If you look at the `create()` factory methods in `JdbcClient`, `RestClient`, and other places within the framework, you will notice that we do not always add a `@return` clause if it only provides redundant information.

Thus, please revert this and related `@return` clause changes.